src/kixi/datastore/metadatastore/geography.clj

```diff
@@ -0,0 +1,19 @@
+(ns kixi.datastore.metadatastore.geography
+  (:require [clojure.spec :as s]))
+
+(def level
+  [:ward :borough :msoa :lsoa :country :region :city])
```

Comment:
Do you consider this to be the complete list or just a partial one (and thus can have more in it)?